projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -855,6 +883,30 @@ function wpcom_launchpad_get_newsletter_subscriber_count() {
 	return (int) $total_subscribers;
 }
 
+/**
+ * Determines if Stripe has been connected.
+ *
+ * @return bool Whether or not Stripe account is connected.
+ */
+function wpcom_is_stripe_connected() {
+	require_lib( 'memberships' );
+	$blog_id  = get_current_blog_id();
+	$settings = (array) get_memberships_settings_for_site( $blog_id );
+	return boolval( $settings['connected_account_id'] );
```

Comment:
Will this output a warning if `$settings['connected_account_id']` is undefined? Should we add an `isset()` wrapper to be super-safe?
```suggestion
	return isset( $settings['connected_account_id'] ) && boolval( $settings['connected_account_id'] );
```

Reply:
Good call. Added the extra checks.